src/routes/resources/resources.routes.ts

```
@@ -262,6 +263,65 @@ export const upvoteQueue = createRoute({
+    ),
+  },
+});
+export const userBookmarks = createRoute({
+  path: "/user/bookmarks",
+  method: "get",
+  tags,
+  responses: {
+    [HttpStatusCodes.OK]: jsonContent(
+      z.array(selectBookmarkSchema),
+      "The User Bookmarks"
+    ),
```

ESLint

[error] Missing trailing comma.

(style/comma-dangle)